doc/source/tutorial/getting_started.ipyml

```diff
@@ -27,6 +27,17 @@ cells:
     __version__
 
 
+- markdown: |
+    <div class="alert alert-warning">
+        The tutorial is generated from Jupyter notebooks which works in "interactive" mode (like in the LArray Editor). 
+        In the "interactive" mode, there is no need to use the function print() to display the content of a variable.
```

Comment:
s/the "interactive"/interactive/
s/the function print()/the print() function
also add a mention of expressions?